modules/custom/az_paragraphs/css/az_paragraphs_full_width.css

```diff
@@ -4,6 +4,10 @@
 .full-width-background {
     max-width: calc(100vw - var(--scrollbar-width) / 2);
 }
+// See https://github.com/az-digital/az_quickstart/pull/1365
+.sidebar_first {
```

Comment:
@trackleft Is this working for you? This CSS file is loading, but I don't see this z-index, overridden or otherwise, in the Firefox CSS inspector. 😕

Reply:
I think firefox doesn't like the comment format

Reply:
Oh, yeah! Firefox shouldn't like it, because that's not valid CSS comment syntax. :-)

I'm wary of being first to re-approve since I keep being wrong... but it's looking right atm...